OWNERS_ALIASES

```diff
@@ -69,3 +69,6 @@ aliases:
     - thunderboltsid
     - yanhua121
     - adiantum
+  assisted-reviewers:
```

Comment:
s/assisted/sno

Reply:
I thought actually about future as we may need not only SNO specific stuff for assisted one day